themes/src/main/resources/theme/base/admin/resources/js/services.js

```diff
@@ -1589,15 +1589,16 @@ module.factory('TimeUnit', function() {
         if (time % 60 == 0) {
             unit = 'Minutes';
             time  = time / 60;
+            if (time % 60 == 0) {
+                unit = 'Hours';
+                time = time / 60;
+                if (time % 24 == 0) {
+                    unit = 'Days'
+                    time = time / 24;
```

Comment:
Is the line `time = time / 24;` needed?

Reply:
I left it because it existed in the original logic, but I removed it because it is certainly unnecessary.